Makefile

```diff
@@ -14,7 +17,7 @@ java:
 	sbt test
 
 objecthash_test: objecthash_test.c objecthash.c
-	cc -Wall -Werror -I/usr/local/include -o objecthash_test objecthash_test.c objecthash.c crypto-algorithms/sha256.c -L/usr/local/lib -ljson-c
+	$(CC) -Wall -Werror $(CPPFLAGS) -o objecthash_test objecthash_test.c objecthash.c $(LDFLAGS) -ljson-c -lcrypto
```

Comment:
Because instead you're forcing me to include some other random SHA256 implementation in the project I want to use objecthash, and the crypto-algorithms implementation is expressly not cryptographically secure and appears to be intended for educational use rather than production.

Also, it seems not entirely unlikely that anyone wanting to use this lib is also going to be using OpenSSL.